src/main.cpp

```diff
@@ -99,3 +81,40 @@ int main(int argc, char *argv[]) {
 
   return (0);
 }
+
+// TODO(JMM): Move this somewhere else?
+void ProcessBoundaryConditions(parthenon::ParthenonManager &pman) {
+  // Ensure only allowed parthenon boundary conditions are used
+  const std::vector<std::string> inner_outer = {"i", "o"};
+  static const parthenon::BoundaryFace loc[][2] = {
+      {parthenon::BoundaryFace::inner_x1, parthenon::BoundaryFace::outer_x1},
+      {parthenon::BoundaryFace::inner_x2, parthenon::BoundaryFace::outer_x2},
+      {parthenon::BoundaryFace::inner_x3, parthenon::BoundaryFace::outer_x3}};
+  static const parthenon::BValFunc outflow[][2] = {
+      {Boundaries::OutflowInnerX1, Boundaries::OutflowOuterX1},
+      {Boundaries::OutflowInnerX2, Boundaries::OutflowOuterX2},
+      {Boundaries::OutflowInnerX3, Boundaries::OutflowOuterX3}};
+  static const parthenon::BValFunc reflect[][2] = {
+      {Boundaries::ReflectInnerX1, Boundaries::ReflectOuterX1},
+      {Boundaries::ReflectInnerX2, Boundaries::ReflectOuterX2},
+      {Boundaries::ReflectInnerX3, Boundaries::ReflectOuterX3}};
+
+  for (int d = 1; d <= 3; ++d) {
+    // outer = 0 for inner face, outer = 1 for outer face
+    for (int outer = 0; outer <= 1; ++outer) {
+      auto &face = inner_outer[outer];
+      const std::string name = face + "x" + std::to_string(d) + "_bc";
+      const std::string parth_bc = pman.pinput->GetString("parthenon/mesh", name);
+      PARTHENON_REQUIRE(parth_bc == "user" || parth_bc == "periodic",
+                        "Only \"user\" and \"periodic\" allowed for parthenon/mesh/" +
+                            name);
+
+      const std::string bc = pman.pinput->GetOrAddString("phoebus", name, "outflow");
+      if (bc == "reflect") {
+        pman.app_input->boundary_conditions[loc[d - 1][outer]] = reflect[d - 1][outer];
+      } else if (bc == "outflow") {
+        pman.app_input->boundary_conditions[loc[d - 1][outer]] = outflow[d - 1][outer];
+      } // periodic boundaries, which are handled by parthenon, so no need to set anything
+    }
+  }
+}
```

Comment:
@brryan I know you did a similar thing in your PR so we should synchronize.

Reply:
Yeah I have a `SetPhoebusBoundaries` function in `src/phoebus_boundaries/phoebus_boundaries.cpp`. I think whichever of our PRs gets merged first, this or `add_mocmc`, the other PR can merge in that change, but ultimately I think this function is best housed in `phoebus_boundaries.cpp`

Reply:
I can move mine to `phoebus_boundaries.cpp`. We should also make sure our implementations are consistent.